### ISSUE TITLE
In application bug reporting

### ISSUE THREAD

**Issue by Armored-Dragon** [overte-org/overte#1136]
It would be convenient to have an in application bug reporting system that will automatically create Github issues when a bug report is submitted. Some users of other platforms have voiced disagreement on having to get off of their platform to report issues. I think it is generally a good idea to make bug reports as easy as possible for users to create.

This feature would require a lot of work before it can be implemented. We would likely need to have some way to filter out duplicate reports from users. We would also need a way to transmit relevant logs while the application is running. There would also be some privacy issues with the log collection as depending on how they are uploaded, those may be available publicly. We would also have to handle potential abuse of the system.

**Comment by JulianGro** [comment]
Having a button to send logs and everything somewhere, or maybe just creating a zip file with relevant data, would be great.
But personally, having in-application bug reporting would be a huge can of worms which seems way overkill. Right now, there is little point because we already get more bug reports than we can fix. But there are also issues like: You need to be logged into GitHub to create issues on GitHub; People need to be able to check if their issue is already reported, which would entail being able to search and browse issues; And low effort bug reports don't really help most of the time, so I am not sure how useful the bug report of a person would be who isn't willing to open the issue tracker in their internet browser.
None of these are showstoppers, but it would mean putting work into a feature which creates more work, while also having more work than anyone can get done.

**Comment by Armored-Dragon** [comment]
Yes this is something for the far far future to consider, but this is something I think is worth considering. You're absolutely right we don't have a need for it now. There'll probably be a time in the next two to three years where it would make sense to have something like this. 
It could be as simple as a helper for Github issues instead of an outright automatic reporter. Just as you suggested, make a zipfile for needed logs, do a sort of courtesy search to see if there was any recent issues that may be relevant to your issue.  I'm not sure how any kind of in-app bug reporting will look in the future, but I still think having something is worth considering.
